var/spack/repos/builtin/packages/unifyfs/package.py

```diff
@@ -16,10 +16,11 @@ class Unifyfs(AutotoolsPackage):
 
     homepage = "https://github.com/LLNL/UnifyFS"
     git      = "https://github.com/LLNL/UnifyFS.git"
-    url      = "https://github.com/LLNL/UnifyFS/releases/download/v0.2.0/unifycr-0.2.0.tar.gz"
+    url      = "https://github.com/LLNL/UnifyFS/releases/download/v0.9.0/unifyfs-0.9.0.tar.gz"
     maintainers = ['CamStan']
 
     version('develop', branch='dev', preferred=True)
+    version('0.9.0', sha256='e6c73e22ef1c23f3141646aa17058b69c1c4e526886771f8fe982da924265b0f')
     version('0.2.0', sha256='7439b0e885234bc64e8cbb449d8abfadd386692766b6f00647a7b6435efb2066')
```

Comment:
If you want to keep this version of the package, you can specify an explicit `url='...'` to download from. Otherwise 0.2.0 will no longer download.

Reply:
@adamjstewart, since we don't currently have a very active user base, we'd prefer to keep users closer to the latest release as a lot of things are changing in the code. With that, I just removed version 0.2.0.